book/src/programmability/collections.md

```diff
@@ -31,3 +29,18 @@ Keys in a `VecMap` are unique, and each key can only be associated with a single
 ```move
 {{#include ../../../packages/samples/sources/programmability/collections.move:vec_map}}
 ```
+
+## Limitations
+
+Standard collection types are a great way to store typed data with guaranteed safety and consistency. However, they are limited by the type of data they can store - the type system won't allow you to store a wrong type in a collection; and they're limited in size - by the object size limit. They will work for relatively small-sized sets and lists, but for larger collections you may need to use a different approach.
```

Comment:
We might want to included that `VecSet` and `VecMap` do not work properly with `==` and `!=`

Reply:
Didn’t know!